build/relsignDSCModules.sh

```diff
@@ -155,10 +155,10 @@ if ! unzip -Z 1>/dev/null; then
 fi
 
 # Working directory used for merging both x64 and x86 versions of a DSC module.
-DscModuleIntermediateDir="/tmp/merging_dsc_modules"
+DscModuleIntermediateDir="tmp/merging_dsc_modules"
```

Comment:
It's not clear where this is run from, so it's not clear where this directory is created. It's important that this is either created in the intermediate directory, or that this directory be added to .gitignore for the project.

I'd like to see documentation on where this should be run from (up top), and I'd like to see verification that you're actually running from there. For example, if the intent is that you're running from the build directory, then verify that you really are running from the build directory.

If the intent is that this is run from the build directory, then these paths definitely need to be changed so they're created in the intermediate directory, or otherwise cleaned up (via make clean/git nuke/whatever).